src/components/SearchBar/SearchBar.styles.ts

```diff
@@ -0,0 +1,19 @@
+import { makeStyles, Theme, createStyles } from '@material-ui/core/styles';
+
+export const useStyles = makeStyles((theme: Theme) =>
+  createStyles({
+    root: {
+      padding: '2px 4px',
+      display: 'flex',
+      alignItems: 'center',
+      width: 400,
```

Comment:
usually when making generic components its best to not set width, unless they have to have width (like checkbox, image, etc). Later the component adapts its width to the parent element and that makes it a lot more versatile. Now its just the layout so we don't care much but a tip for the future